kss/builder/decanter/scss/_content.scss

```diff
@@ -131,7 +131,6 @@
   clear: left;
   border: 2px dashed transparent;
   position: relative; // Contain the example's absolute positioning.
-  z-index: 0; // Establishes a local stacking context.
```

Comment:
Why was this removed?

Reply:
Yeah, I removed it because it caused the main nav and the other component beneath to overlap each other in the styleguide. It doesn't seem to cause problems elsewhere so far.